assets/airlines/bos.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Not sure what happened with the formatting here. Also add new line at the end of the file

Reply:
done. musta been a vscode thing. still getting adjusted... it definitely has its pluses and minuses to say the least :)